analysis_options.yaml

```
@@ -1 +1 @@
-include: package:very_good_analysis/analysis_options.2.4.0.yaml
+include: package:very_good_analysis/analysis_options.yaml
```

```suggestion
include: package:very_good_analysis/analysis_options.3.0.2.yaml
```

thoughts on being explicit about the version? In the past always using the latest version led to spontaneous pipeline failure if a new lint rule was added. This the developer is in control over when to upgrade.

---

The standard way of controlling the version of a dependency is the pubspec file. 

Considering that, it may be counter intuitive to update the analysis option dpeendecy having to change it in two places. 

 If we want to pin o a version we could remove the caret from the pubspec entry.

---

The difference is technically the package can contain things other than lint rules (the deprecated `unawaited`). Developers may want to opt into all non-breaking changes for functionality exported by the library while still locking the version of analysis_options being applied. Wdyt?